judo-ui-react/src/main/java/hu/blackbelt/judo/ui/generator/react/UiPageHelper.java

```diff
@@ -214,6 +214,13 @@ public static String getPageRoute(PageDefinition page) {
         return route;
     }
 
+    public static List<Action> getButtonActions(PageDefinition page) {
+        return page.getButtons().stream()
+                .map(b -> ((Button) b).getAction())
+                .filter(a -> !(a instanceof BackAction))
+                .collect(Collectors.toList());
```

Comment:
FYI: since Java 16 there is a `toList()` function for Streams (which is unmodifiable compared to Collectors.toList() which returns a modifiable list).

Reply:
Will do it that way next time.